.gitignore

```diff
@@ -59,3 +59,8 @@ __pycache__/
 
 # android build
 nntrainer_for_android.tar.gz
+
```

Comment:
```suggestion
# gtest
test/jni/googletest/
```

could we also add googletest source? ([ref](https://github.com/nnstreamer/nntrainer/blob/21b300a8553c3fbc18ca1bf492e4d18477e81104/test/jni/README.md))